spec/features/groups/accept_invitation_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Students Joining Groups' do
+  context 'accepting invitation' do
```

Comment:
Why isn't this the top-level description? It seems silly unless you want to group it with sending invitations?